### PR TITLE
DC-641 Update Custom VCC API docs

### PR DESCRIPTION
**Pull request by eltesoro-koji**
Updates to readme and package.json, including upping the version to 0.0.11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Koji Custom VCC SDK
 ![npm (scoped)](https://img.shields.io/npm/v/@withkoji/custom-vcc-sdk?color=green&style=flat-square)
 
-**Library for creating custom controls to use in Koji templates.**
+**Library for creating custom controls to use in Koji app.**
```
Comment (rasienko) [inline README.md:4]:
```suggestion
**Library for creating custom controls to use in Koji apps.**
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 ## Overview
 
-The @withkoji/custom-vcc-sdk package enables you to implement custom controls for capturing user input on the frontend of your Koji template.
-With this package, you can provide customizations that match closely with the application you are developing.
-For example, some Koji templates provide tile map editors, sound enhancers, or custom avatar creators to enhance the interactivity for remixers.
+The @withkoji/custom-vcc-sdk package enables you to implement custom controls for capturing user input on the frontend of your Koji app.
+With this package, you can provide customizations that is better suited for the application you are developing.
```
Comment (rasienko) [inline README.md:9]:
```suggestion
With this package, you can provide customizations that are better suited for the application you are developing.
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+For example, some Koji apps provide tile map editors, sound enhancers, or custom avatar creators to enhance the interactivity in the app.
 
 ## Installation
```
Comment (rasienko) [inline README.md:13]: Did we want to remove `--save` from the installation comand?

File: README.md [hunk 4]
```diff
@@ -55,10 +55,10 @@ customVcc.save();
 
 ### Using custom controls
 
-To make a custom control accessible to Koji templates, you must serve it at a custom domain.
+To make a custom control accessible to Koji app, you must serve it at a custom domain.
```
Comment (rasienko) [inline README.md:58]:
```suggestion
To make a custom control accessible to other Koji apps, you must serve it at a custom domain.
```

File: README.md [hunk 5]
```diff
@@ -55,10 +55,10 @@ customVcc.save();
...
 After you publish the project, add a subdomain under the `koji-vccs.com` root domain.
 
-Then, you can use the control in a Koji template in one of these ways.
+Then, you can use the control in a Koji app in one of these ways.
 
 * (Recommended) With the @withkoji/core package, call the `Koji.ui.capture.custom` method with the subdomain name. For example:
```
Comment (rasienko) [inline README.md:64]: Updates needed to related resources: 
- Update link and title of "Building your first custom VCC" to point to the current page ("Building custom controls")
- Remove Custom VCC blueprint (that page has been deprecated)

**Review by rasienko — CHANGES_REQUESTED** [review]
@eltesoro-koji A couple of small fixes and then this should be ready to go. Thanks!

File: README.md (hunk 1 above)
Comment (eltesoro-koji) [reply README.md:4]: Fixed.

File: README.md (hunk 2 above)
Comment (eltesoro-koji) [reply README.md:9]: Fixed.

File: README.md (hunk 4 above)
Comment (eltesoro-koji) [reply README.md:58]: Fixed.

File: README.md (hunk 3 above)
Comment (eltesoro-koji) [reply README.md:13]: Fixed.

File: README.md (hunk 5 above)
Comment (eltesoro-koji) [reply README.md:64]: Fixed.

**Review by rasienko — APPROVED** [review]

**Comment by rasienko** [comment]
LGTM! @sthielen can you merge and push to npm when you approve?

**Pull request merged**